### PR TITLE
fix(dev-server) cloudflare adapter restart

### PR DESCRIPTION
**Pull request by komapotter**
* https://github.com/honojs/vite-plugins/issues/172

### CODE REVIEW

File: packages/dev-server/src/adapter/cloudflare.ts [hunk 1]
```diff
@@ -41,12 +43,16 @@ export const cloudflareAdapter: (options?: CloudflareAdapterOptions) => Promise<
...
-         */
+      if (proxy !== undefined) {
+        try {
+          await proxy.dispose()
+        } catch (error) {
+          /**
+           * It throws an error if server is not running.
+           */
+        } finally {
+          proxy = undefined as Awaited<ReturnType<typeof getPlatformProxy<Env>>> | undefined
```
Comment (yusukebe) [inline packages/dev-server/src/adapter/cloudflare.ts:54]: You don't have to write the `as`. It will be:

```ts
proxy = undefined
```

**Comment by yusukebe** [comment]
Hi @komapotter 

Thank you for the PR. Almost all are good. I've left a comment. Please check it.

File: packages/dev-server/src/adapter/cloudflare.ts (hunk 1 above)
Comment (komapotter) [reply packages/dev-server/src/adapter/cloudflare.ts]: Thank you for checking! I have made the corrections.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@komapotter 

Looks good! Let's go with it. Thank you!

**Pull request merged**